### PR TITLE
[W.I.P] [V1.3] Add the Mobilenetv1 case for calibration tool UT.

### PR DESCRIPTION
**Pull request by guomingz**
1.Add the Mobilenetv1 case for calibration tool UT;
2.Code clean.

### CODE REVIEW

File: python/paddle/fluid/contrib/tests/test_calibration.py [hunk 1]
```diff
@@ -117,26 +117,36 @@ def val(data_dir=DATA_DIR):
...
+        cmd = 'rm -rf {} {} && mkdir {} && wget {} && tar xf {} -C {}'.format(
             resnet50_unzip_folder_name, resnet50_zip_name,
             resnet50_unzip_folder_name, resnet50_fp32_model_url,
             resnet50_zip_name, resnet50_unzip_folder_name)
         os.system(cmd)
 
-        self.iterations = 100
-        self.skip_batch_num = 5
+        # mobilnet fp32 data
+        mobilenetv1_fp32_model_url = 'http://paddle-inference-dist.bj.bcebos.com/int8/mobilenetv1_int8_model.tar.gz'
```
Comment (luotao1) [inline python/paddle/fluid/contrib/tests/test_calibration.py:140]: - 下载地址请都改成：http://paddle-inference-dist.cdn.bcebos.com/
即bj-》cdn，用来加速。

File: python/paddle/fluid/contrib/tests/test_calibration.py [hunk 2]
```diff
@@ -225,6 +235,13 @@ def test_calibration_for_resnet50(self):
         delta_value = np.abs(fp32_acc1 - int8_acc1)
         self.assertLess(delta_value, 0.01)
 
+    def test_calibration_for_mobilenetv1(self):
+        fp32_acc1 = self.run_program("mobilenetv1_fp32/model")
+        self.run_program("mobilenetv1_fp32/model", True, algo='KL')
+        int8_acc1 = self.run_program("calibration_out")
+        delta_value = np.abs(fp32_acc1 - int8_acc1)
+        self.assertLess(delta_value, 0.01)
```
Comment (luotao1) [inline python/paddle/fluid/contrib/tests/test_calibration.py:243]: @hshen14 Please use two TestClass, if one of them is failed, we will easily find which one is failed on CI log.
```
class TestCalibrationForResnet50(unittest.TestCase):
   ...
class TestCalibrationForMobilenetv1(TestCalibrationForResnet50):
   ...
```

Comment (hshen14) [reply python/paddle/fluid/contrib/tests/test_calibration.py:243]: Done.

**Comment by luotao1** [comment]
close due to #15539

**Pull request closed without merging**